app/src/main/java/de/danoeh/antennapod/adapter/PlaybackStatisticsListAdapter.java

```diff
@@ -1,6 +1,8 @@
 package de.danoeh.antennapod.adapter;
 
 import android.content.Context;
+import android.view.View;
+
```

Comment:
Please revert the lines above. Not touching unrelated code keeps the git history clean.